### DISCUSSION TITLE
Instagram reels/videos supported by Fancybox?

### DISCUSSION THREAD

**Discussion by 06tech** [fancyapps/ui#659] [Q&A]
I am trying to use Fancybox to embed Instagram videos and it fails (works okay for YouTube and Vimeo) - does anyone have an example of using Fancybox with Instagram?

Thanks in advance

**Comment by fancyapps** [comment]
If you were expecting it to work similarly to Youtube and Vimeo links, then sorry, no. I do not think it is possible to create embeddable links for instagram.
